### PR TITLE
ignore embeddings in extraction process

### PR DESCRIPTION
**Pull request by lauritowal**

### CODE REVIEW

File: elk/extraction/extraction.py [hunk 1]
```diff
@@ -192,11 +191,9 @@ def extract_hiddens(
         template_path=cfg.template_path,
         rank=rank,
         world_size=world_size,
-        seed=cfg.seed,
```
Comment (AlexTMallen) [inline elk/extraction/extraction.py:195]: This seed is used in load_prompts, why was it removed?

**Review by AlexTMallen — COMMENTED** [review]
I made a couple changes. Most importantly, the changes to the handling of labels fixes extraction on deberta. Waiting for another review.

File: elk/extraction/extraction.py (hunk 1 above)
Comment (lauritowal) [reply elk/extraction/extraction.py:195]: @AlexTMallen what do you mean, it is still there or not?

**Review by norabelrose — APPROVED** [review]
LGTM with Alex's changes

**Pull request merged**